docs/gallery_code/general/plot_anomaly_log_colouring.py

```diff
@@ -12,18 +12,15 @@
 "zero band" which is plotted in white.
 
 To do this, we create a custom value mapping function (normalization) using
-the matplotlib Norm class `matplotlib.colours.SymLogNorm
-<https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.SymLogNorm.html>`_.
-We use this to make a cell-filled pseudocolour plot with a colorbar.
+the matplotlib Norm class :doc:`matplotlib:api/_as_gen/matplotlib.colors.SymLogNorm`.
```

Comment:
This is still pretty specific to the layout; you can link :obj:`matplotlib.colors.SymLogNorm` for the thing itself.

Reply:
I agree, closer to being layout agnostic, but not quite there yet... I'll try your suggestion, thanks.

Reply:
Neat. That seems to work @QuLogic :+1: 

That's much better, thanks.